arch/arm/src/samv7/Kconfig

```diff
@@ -3003,4 +3003,51 @@ config SAMV7_MCAN_REGDEBUG
 
 endmenu # CAN device driver options
 endif # SAMV7_MCAN
+
+menu "QEncoder Driver"
+	depends on SENSORS_QENCODER
+	depends on SAMV7_TC0 || SAMV7_TC1 || SAMV7_TC2 || SAMV7_TC3
```

Comment:
Please use SAMV7_HAVE_TC instead of each timer

Reply:
Done